src/pybind/mgr/cephadm/serve.py

```diff
@@ -929,6 +931,7 @@ def _check_daemons(self) -> None:
                     dd.name()))
                 action = 'reconfig'
             elif last_deps != deps:
+                self.log.error(f"DEBUG: last deps is different for {dd.name()} will reconfig")
```

Comment:
This is a draft PR, but we shouldn't forget this.

Reply:
thanks. will remove!